portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -362,6 +422,30 @@ definition {
 		}
 	}
 
+	@description = "LPS-178858 Confirm that the modal data set view is displayed"
+	@priority = 5
+	test CanAssretTheDataSetViewCanBeCreate {
```

Comment:
Fix the name of the test, it should be CanAssertTheDataSetViewCanBeCreate

Reply:
@CarlosBrichete what do you think?